games/src/mathemagic/mod.rs

```diff
@@ -14,14 +14,12 @@ atlas!(atlas, "games/src/mathemagic/terrain_atlas.png", 32);
 
 sprite!(protagonist, "games/src/mathemagic/lidia.png", 576);
 
-const _: &[u8] = include_bytes!("../../assets/slime/slime_monster_spritesheet.png");
```

Comment:
This was required before because otherwise Rust wouldn't recompile if the file read by the proc macro changed.

Reply:
I would just do a cargo clean in such a case. I thought this line was unnecessary since spritesheets _are_ included.